OWNERS_TEMPLATE.md

```
@@ -0,0 +1,145 @@
+
+**Do not manually add OWNERS to automated charts** in multiclusterhub-operator - they will be removed on the next regeneration if they don't exist upstream.
+
+**Exception:** Charts manually maintained in multiclusterhub-operator (not in automation config) can have manual OWNERS files - the automation won't touch them.
+
+## Where to Add OWNERS File
+
+### For Helm-based Components
+
+Add the OWNERS file to your chart directory in your component repository:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)